### DISCUSSION TITLE
How to use syslog?

### DISCUSSION THREAD

**Discussion by Pacjunk** [lora-aprs/LoRa_APRS_iGate#227] [Q&A] [answered]
Hi, I am trying to use syslog for logging of the igate but cannot get it to work. I have my own syslog server setup and I have tested it and it is working correctly. I change the settings in the json file to point to this syslog server and nothing is logged. What am I doing wrong? Does anyone else have it working?

I have looked at the doco here, but the syslog configuration is not described.
Thanks,

**Comment by matix2120** [comment]
Check if you're able to receive any messages on the server at all:
Run on server: nc -u -l <port>
And configure ip and port accordingly in json.

**Reply by Pacjunk** [reply]
I have tested from a couple of different linux boxes and the syslog server receives the messages. It just doesn't receive anything from the igate.  I was asking to see if anyone else had it working.

**Comment by Pacjunk** [comment, marked as answer]
It seems to be something with the syslog server on my QNAP NAS. If I use rsyslog on a linux box it works.
